### ISSUE TITLE
Floats parsed incorrectly

### ISSUE THREAD

**Issue by zaibacu** [jeremiah-shaulov/nop-json#7]
If I have JSON like this:

```json
[{"value": 1}, {"value": 2}, {"value": 3}, {"value": -3000.0}, {"value": -1000}, {"value": -1.0}, {"value": 100.0}]
```

and print every value inside array, this is what I get:
```
Value: {"value":1}
Value: {"value":2}
Value: {"value":3}
Value: {"value":-0.}
Value: {"value":-1000}
Value: {"value":-0.}
Value: {"value":0.1}
```

**Comment by jeremiah-shaulov** [comment]
Hi. I fixed this bug in v2.0.5

**Comment by zaibacu** [comment]
Yes, seems to be working OK now
